Homework1/Sorting/Sorting/Program.cs

```diff
@@ -0,0 +1,58 @@
+﻿namespace Sorting;
+
+using System;
+
+/// <summary>
+/// Class containing BubbleSort and Main
+/// </summary>
+class Program
+{
+    /// <summary>
+    /// Sorts an array using Bubble sort
+    /// </summary>
+    /// <param name="array">Array for sorting</param>
+    public static void BubbleSort(int[] array)
+    {
+        for (var i = 0; i < array.Length - 1; ++i)
+        {
+            for (var j = 0; j < array.Length - 1 - i; ++j)
+            {
+                if (array[j] > array[j + 1])
+                {
+                    int temp = array[j];
+                    array[j] = array[j + 1];
+                    array[j + 1] = temp;
```

Comment:
Кстати, можно как в Питоне:
```suggestion
                    (array[j], array[j + 1]) = (array[j + 1], array[j]);
```